lib/robots/dor_repo/gis_assembly/assign_placenames.rb

```diff
@@ -49,18 +48,6 @@ def add_uri_to_subject(subject, uri)
             uri: 'http://www.geonames.org/ontology#'
           }
         end
-
-        def coverage_subjects_for(value)
```

Comment:
Per Kim, we should no longer create these duplicate subjects with type "coverage" since they are just duplicates of existing subjects of type "place".